wrangler/src/txFunctions/run.js

```diff
@@ -45,6 +49,11 @@ export default async ({ request, params, env }) => {
   const txFeesId = TX_FEES.idFromName(authedPublicKey)
   const txFeesStub = TX_FEES.get(txFeesId)
 
+  if (singleUse) { // If auth token is single use check if it's already been used
+    await txFeesStub.fetch(`/${authedHash}`, {method: 'POST'}).then(handleResponse)
```

Comment:
does this throw if the token exists?

Reply:
Looks like method with return 402, I guess its unclear to me what that results in here

Reply:
Yeah, if the auth token is single use it fires the endpoint which will look for the existence of the token in `this.storage`. If it finds it it will throw in the durable object and bubble back to the worker. If it doesn't exist (ala hasn't been used yet) it will be added to the DO with a response of OK back to the worker which will continue with it's execution